src/manifest.rs

```diff
@@ -49,21 +49,15 @@ fn read_cargo_toml(path: &str) -> Result<CargoManifest, Error> {
 }
 
 impl CargoManifest {
-    fn into_npm(self, scope: Option<String>) -> NpmPackage {
+    fn into_npm(mut self, scope: Option<String>) -> NpmPackage {
```

Comment:
can we do it without this mutation? 
replacing the `if` with a `match | if.else` will help right?